ui/packages/gallery/src/Gallery.svelte

```diff
@@ -151,7 +163,7 @@
 	<div
 		bind:clientHeight={height}
 		class="grid-wrap"
-		class:fixed-height={style.height !== "auto"}
+		class:fixed-height={!style.height || style.height == "auto"}
```

Comment:
Before, fixed-height would always be set if the user specified a height value in `style`. This was restricting the size of the div containing the gallery.

Now it's only set if `height` is not set, or if it's auto.